include/engine/polyline_compressor.hpp

```diff
@@ -13,14 +15,42 @@ namespace engine
 namespace detail
 {
 constexpr double POLYLINE_PRECISION = 1e5;
```

Comment:
This exists for use by the decodePolyline function. I haven't changed decode function to make it customizable. Should I? I couldn't figure out if I should because the only place that decodePolyline is used is here: https://github.com/Project-OSRM/osrm-backend/blob/master/include/server/api/base_parameters_grammar.hpp#L127 and I'm not sure what that does.